### PR TITLE
@acjay: Fixes staging flag.

### PR DESCRIPTION
**Pull request by ashfurrow**

### CODE REVIEW

**Comment by acjay** [comment]
CI still busted?

**Comment by ashfurrow** [comment]
Yes, tracked in Trello.

**Comment by ashfurrow** [comment]
Oh weird, hold up.

File: Kiosk/Auction Listings/ListingsViewController.swift [hunk 1]
```diff
@@ -44,7 +44,7 @@ class ListingsViewController: UIViewController {
 
         // Set up development environment.
 
-        if APIKeys.sharedKeys.stubResponses {
+        if APIKeys.sharedKeys.stubResponses && AppSetup.sharedState.isTesting == false {
```
Comment (acjay) [inline Kiosk/Auction Listings/ListingsViewController.swift]: As a future refactor thought, if this logic isn't strictly specific to the view, I like the idea of encapsulating it in a "config model" of some sort. That way, if anything else needs to perform a similar mode check, the actual logic is all in one spot.

Comment (ashfurrow) [inline Kiosk/Auction Listings/ListingsViewController.swift]: Agreed – this was just to get things fixed now, will follow-up.

**Comment by acjay** [comment]
Holding up! Let me know when I can press the button

**Comment by ashfurrow** [comment]
OK, tests passing locally. Had to change a few things. Agreed we need a better abstraction, but this will do for now. Merge when ready :+1:

**Comment by ashfurrow** [comment]
Ha, nice. The deploy got rolled into the PR.

**Pull request merged**